apps/mochi-web/types/mochi-pay-schema.ts

```diff
@@ -405,11 +437,17 @@ export interface ViewApplicationStatsResponse {
   data?: ViewApplicationStats
 }
 
+export interface ViewDataResponse {
+  data?: any
```

Comment:
Don't we have the full interface for the data?

Also, is this schema generated or do you manually input it?